api.js

```diff
@@ -171,9 +171,9 @@ class Api extends EventEmitter {
 		const execArgv = this.options.testOnlyExecArgv || process.execArgv;
 		let debugArgIndex = -1;
 
-		// --debug-brk is used in addition to --inspect to break on first line and wait
+		// --inspect-brk is used in addition to --inspect to break on first line and wait
 		execArgv.some((arg, index) => {
-			const isDebugArg = arg === '--inspect' || arg.indexOf('--inspect=') === 0;
+			const isDebugArg = /^--inspect(-brk)?($|=)/.test(arg);
```

Comment:
There's no `--inspect-brk`. Could you update t=your regex removing that validation, please?